### ISSUE TITLE
AlgGeo II typos

### ISSUE THREAD

**Issue by FlorianAdler** [Nicholas42/AlgebraFranke#25]
Remark on the bottom of p. 27: X and Y should be swapped.

**Comment by BastiaanCnossen** [comment]
For convenience, I will just use this same issue for sharing some more typos I came across in the AlgGeo II script. I also added a few suggestions about improving the structure of chapter 3.
- p. 4, cor. 1.1.1: we have coker(f)(U) with local sections in U instead of global sections in X.
- p. 5, prop. 1.1.4: the map i^* in J = ker(i^*) should go to i_*O_X instead of i_*O_Y
- p. 10, example 1.2.1(b): in the examples you give, you swapped the alternating Cech-complex with the 'unalternating' one.
- p. 17, 'Professor Franke run out of primes' should either be 'runs' or 'ran'.
- p. 20, 'Then we get Spec R \to X' should have reversed direction: p: X \to Spec R.
- p. 22, The word 'fpr' should be 'for'.
- p. 24, near the bottom of the page. Here Franke used a 'U' in the statement of the theorem, and then assumes without loss of generality that this is Y. After it, he uses the U for something else. I think it's better to find a way around this so that U never gets a double meaning.
- p. 25, top of the page: You want to take global sections of R^pf_*M in equation (2).
- p. 26. When you write down the more general form of the Leray Spectral Sequence, is this meant to be on global sections?
- p. 27. As mentioned above, X and Y should be swapped.
- p. 31, bottom of the page: when you take the intersection of curly U with V at some point, this V should be one of the U_i. You want that the affine open covering (curly U) \cap U_i has U_i itself in it for the argument to work. 
- p. 36, Definition 2.1.3: You remark that Franke forgot about the exception m = 0, but it seems to me that with the usual convention of [n]^0 = {*} (empty product being a final object) everything goes fine as stated. But maybe I'm missing the same thing as Franke was missing...
- p. 39 This is actually not a typo. I didn't get in the remark after definition 2.1.4 why the chain of isomorphisms proving an isomorphism of two direct sums implies that the summands are already isomorphic. I guess this would follow from the explicit form of the isomorphism finally obtained between the two direct sums, being induced by the two canonical isomorphisms for P and Q, and by the two zero-maps?
- p. 45, near top of the page. You formulated 'Hence the tensor product over arbitrary preschemes preserves quasi-coherentness'. Maybe it is good to explicitly mention that the tensor product of two q.c. R-modules M and N over _any_ q.c. sheaf of O_X-algebras R (hmmm maybe even any q.c. sheaf of rings?) is again quasi-coherent.
- p. 47, top of the page: the S appearing here should just be an R.
- p. 48, top of the page. In anticipation on how the letters A and B will be used eventually, it might be advisable to swap the roles of A and B in the discussion at the top of the page and consider a ring morphism B \to A instead of the other way around (or to use different letters.) Now you end up with the equation \phi_B(\alpha(g)) = \alpha(\phi_A(g)), and directly after that you use it in the form \phi_A(\alpha(g)) = \alpha(\phi_B(g)) without explicitly repeating.
- p. 48, second paragraph: note that the argument needed for showing that p is affine also relies on the fact that the P^n_A\V(X_i) cover P^n_A.
- p. 49, remark 2.2.4 (a). I don't entirely get why you deviated from Franke's notation and changed m into -m. It is unusual to use the letter 'm' for a strictly negative integer. Writing 'L ^{-m} = (L^-1)^m' is a lot more suggestive, since by the way it is written one directly understands that the 'm' is to be taken positively. Also the '-m' in the power now has the strong association of being a negative power, which is exactly the opposite of what you wanted to achieve with the remark.
- p. 50 Lemma 2.2.2(a): There is a global sections (X) missing after L^k \otimes M.
- p. 50, proof of lemma 2.2.2, in the part 'For (b), ...'. What you meant is that \mu_i is taken such that lambda^l \otimes m = \mu_i on `U_i \setminus V(\lambda)` instead of on U_i. 
- p. 52, You refer to 'this footnote' and use a link that can be clicked on in the pdf-version. Since many people print out your notes, such references won't be clear and you might want to consider to give a brief indication of where the footnote was placed. 
I think this also happened a few other times in the script; not sure though.
In this case, the whole reference to the footnote might even be replaced by a reference to one of the remarks that Franke made about pull-backs a few weeks later.
- p. 56, Theorem 4: you mean 'and R_1 if a finitely generated R_0-module' (not an R-module).
- p. 68, near top of page: in fact, you don't only want that it suffices to show that Proj R \to Spec A is closed, but that it even suffices to show that the image of Proj R in Spec A is closed (this is the only thing you show in the end.)
- p. 68, middle of page: you mean that for infinitely many j, R_j \otimes_A k(p) _is not equal to zero_. (That part is missing now.)
- p. 68, Above theorem 5: there is a space missing in 'generatedover'. 
- p. 69, top of page. Here you mean 'in the sense that L^{k} is generated by global sections', not L. In the lecture, Franke did indeed write 'L' there, but he defined 'very ampleness' explicitly for a line bundle L (although admittedly the middle of some other sentence wasn't the best place for him to define this....) 
- p. 70, like I mentioned before, the two references 'here' you're making won't help the reader that printed the document and a short explicit reference would be helpful.
- p. 70, it wasn't a priori clear to me why any line bundle on X is isomorphic to a line bundle of the form O(D). I know that Franke just completely went over this and didn't prove it, but if you turn out to have some extra time, then this could be something worth clarifying.
- p. 70, bottom of the page. Franke defined the degree of a divisor D as the sum on the right of the bottom equation, not as the degree of the line bundle  O(D). This is also what you use later in fact 3.2.3 (i.e. you use that increasing/decreasing the coefficient of one of the closed points in the divisor D by 1 changes its degree by plus/minus 1.)
- General comment about this chapter: Franke made the structure of this chapter horrible in my opinion. The order he chose to treat the proofs is definitely not the most enlightening.
- Anyway, that being said, I would like to point out some points where some extra explanation or some reordering of the material might be helpful. (I'm sorry if I just have an old version of your script and you turned out to already have rewritten chapter 3... In that case, just feel free to ignore the next few comments.)

1. First, as you point out in remark 3.2.1 on page 74, it is not immediately clear at all why the dimension of H^1(C,V) should be the same as the dimension of Hom(V,\Omega_C) when we assume Serre Duality to be proved. I'm greatly thankful for your explanation there!!!! I would advice to place this remark directly after Franke states this equality in theorem 8 on page 71, to provide the reader directly with this explanation.
2. Second, the equation (5) on page 72 is obtained by using this equation (4) and thus already assumes that we have proven Serre Duality. In other words: the proof that \chi(C,O_C) = 1-g needs Serre Duality. After that, the process of proving Riemann Roch is proving the equation (1) on page 73, and you refer to 'as pointed out, this will be enough'. My advice would be to spell this out, maybe even make a separate lemma out of it, like: 

_Lemma_: For proving Theorem 7(a), it is sufficient to show that \chi(C,O(D)) = deg D + \chi(C,O_C). 
Furthermore, for proving Theorem 8, it is sufficient to show that we have \chi(C,O_C) = 1-g. For this, theorem 7(b) and (c) suffice.
_Proof_: The first part is obvious, since all line bundles have the form O(D) as remarked before, so setting deg(O(D)) = \chi(C,O(D)) - \chi(C,O_C) will work. For the second part, note that we can do (insert your argumentation about the filtration of a vector bundle V). 
Finally, note that Serre Duality does indeed imply \chi(C,O_C) = 1-g because (add here the calculation given before of dim O_C(C)  = 1 and also add the calculation dim H^1(C,O_C) = dim H^0(C,\Omega_C) = g where we explicitly see how this follows from Serre Duality.)

3. Third, In fact 3.2.3 on page 75, I'd recommend repeating equation (1). After the lemma, it might also be nice to explicitly state that for proving both Theorem 7(a) and Theorem 8, it now only remains to prove Theorem 7(b) and 7(c) by the lemma given above. This adds some more structure to the chapter and makes clear how far we are in proving the theorems. It also explains why in fact 3.2.4 we only prove theorem 7(b) and 7(c).
4. In fact 3.2.4, maybe you can put a little more emphasis on the proof for theorem 7(c). It's now partially hidden in the proof of theorem 7(b). Putting this completely in the beginning after remarking that the identification of \Omega_{P^1} with O(-2) is enough for Theorem 7(b) will then capture the whole idea of the proof in the first paragraph and it is clear that in fact all of theorem 7 and 8 follow after this identification, and not only part 7(b). After this, the reader will know that all what follows are just technical computations to show that this identification does indeed hold.

- p. 76, third paragraph or something. Maybe it is good to mention that Franke does indeed use here that deg(O(1)) = 1, like you already predicted he would on page 75. 
- p. 79 middle of the page: the 'hhe' should be 'the'. Also the 'achive' should be 'achieve'.
- p. 85, first equation of the page: The second Hom should be over O_C, not over O_{P^1_k}.
- p. 85, 'It is sufficient that this isomorphism satisfies Theorem 7(a)', aren't we talking about Theorem 7(b) here?
- p. 85, just before the remark: Maybe you can refer explicitly to proposition 3.3.4, which seems to me what you use here.

This was what I came across so far. In the next few days, a few more typos might be added once I read some of the parts of the script that I skipped for now (in particular chapter 4, perhaps also the appendix.)

I wish you and all others good luck with your exam(s)!
Best,
Bastiaan

**Comment by BastiaanCnossen** [comment]
Some additional remarks/typos:
- p. 48. At the bottom, in the footnote, you use the notation g(X_i). Do we see X_i here as the (i+1)-th basis-vector of A^{n+1}, and g(X_i) as an expansion in the {X_j} given by g applied to this vector?
- p. 61. In corollary 2.3.2, I would recommend giving the isomorphism Proj(tau^* R) \to P x_S T a separate line to give it more emphasis, so that it will directly catch the eye. Also I would still write out the P_T as Proj(tau^* R) in that line. It makes lemmas as this more readible, since with one quick look you can recall what the lemma says.
- p. 61, corollary 2.3.2. In addition to the above, since the definition of \epsilon_k isn't that long, you might want to consider to explicitly repeat it. It takes some searching to find the previous definition right now.
- p. 64. At the bottom, how do we see surjectivity of this map R_n \to f_nO_Y(n)? Is this supposed to be trivial or did we already see something like this before somewhere?
- p. 65 Just a stupid style suggestion in the second paragraph, but would it make sense to use a tilde in I_X to already imply by the notation that it corresponds to \tilde{X}?
- p. 67. Again a stupid style suggestion, but before you state proposition 2.4.1, you could replace 'lemma 2.3.4 gives...' with 'combined with lemma 2.3.4, this gives...' since the proposition contains several results that don't have anything to do with that lemma.
- p. 67. In proposition 2.4.2, the Proj(X) should be Proj(R).
- p. 68. After the statement of Theorem 5, it would maybe be nice to refer the reader to Chow's lemma (not necessarily going into detail) so the reader gets the idea of the proof. 
- p. 74. In the proof of fact 3.2.2, you write at some point "Shrinking U if necessary, we may ...". This sentence seems to be a remainder of an earlier version of the proof and should either be relocated or be removed.
- The footnote you have at page 78 seems to be necessary for the conclusion of fact 3.3.1 at page 77 as well, isn't it? Since Franke seems to use it several times, perhaps you can explicitly insert this in the explanation preceding fact 3.3.1?
- p. 79. In lemma 3.3.2, Franke wrote xi - xi(c_i), but I don't get how we regard xi as an element of m^2_{C,c_i}. Do we 'evaluate' at the generic point or something?
- p. 86. In the proof of fact 4.1.1, there are brackets missing in Spec(R \otimes_A B), making it look as Spec(R) \otimes_A B. 
- p. 88. You repeated the 'V' in the first line.
- p. 88. Maybe you can deviate from the notation Franke used and let G explicitly depend on V

I hope I won't have scared you too much with these two long lists of remarks XD

**Comment by FlorianAdler** [comment]
It took some time, but now I'm through with your list. I followed most of your suggestions, which were excellent as usual. Some comments:

> p. 26. When you write down the more general form of the Leray Spectral Sequence, is this meant to be on global sections?

- This is a spectral sequence of sheaves; I added a comment on that.

> p. 48. At the bottom, in the footnote, you use the notation g(X_i). Do we see X_i here as the (i+1)-th basis-vector of A^{n+1}, and g(X_i) as an expansion in the {X_j} given by g applied to this vector?

- I think that's basically it. I imagined g acting on the polynomial ring by sending X_i to some homogeneous linear polynomial l_i.

> p. 68. After the statement of Theorem 5, it would maybe be nice to refer the reader to Chow's lemma (not necessarily going into detail) so the reader gets the idea of the proof.

- I added a proof of Theorem 5 (and also of Chow's lemma and the theorem on formal functions).

> General comment about this chapter: Franke made the structure of this chapter horrible in my opinion. The order he chose to treat the proofs is definitely not the most enlightening.

- Section 3.1 and 3.2 are completely restructured by now. I hope, this makes the Chapter more readable.

> p. 79. In lemma 3.3.2, Franke wrote xi - xi(c_i), but I don't get how we regard xi as an element of m^2_{C,c_i}. Do we 'evaluate' at the generic point or something?

- I added a comment on that (and hope I got it correctly).

Again, thanks a lot. Your comments really help improving the quality of these notes.

**Comment by bforras** [comment]
As the list typos for the AG2 script seems to be concentrated in this issue, I‘ll just list the ones I found here. (If I should have opened a new issue, I wholeheartedly apologise.)

* You may wish to consider using the word noetherianity instead of noetherianness as it seems to be more prevalent based on number of Google search results. Similarly for abelianity vs. abelianness, although the number of search results don‘t differ significantly in their case. Similarly for projectivity vs. projectiveness and (quasi-)coherence vs. (quasi-)coherentness.
* Before Proposition 1.1.4: unnecessary “to”.
* Proof of Definition 1.2.2: the map s_\ell goes from the (n – 1)th group to the nth one. Moreover, in the equation defining t_\ell\psi there should be i_n instead of i_{n-1} on the right hand side.
* Example 1.2.1(c): more generally
* Lemma 1.2.1(a): (\mathcal W, \mathcal U) should be (\mathcal W, \mathcal V)
* Proposition 1.2.2: an affine open cover
* Lemma 1.2.2(b) overloads i in a disturbing way (at least upon first glance).
* Proof of Lemma 1.2.2(b): you seem to write QCoh everywhere but here.
* Proof of Proposition 1.5.1: with the above simplifications
* Section 2.3, equation of the structure shead: Y should be P
* Fact 2.3.2: replace “and” by “then”.
* Example 2.3.2: I believe the reference should be to AG1, Example 2.6.2.
* Definitions 1.1.6 and 2.4.1: “it it” should be “if it”
* Thoughout Section 3 there are quite a few instances where sheaf-Hom should be used instead of Hom.
* Lemma 3.3.2(a) replace D by d
* Just before Proposition 3.4.1: Let $C$ always be
* Remark after Definition A.1.4, part (b), second paragraph: finite products and finite coproducts
* Just before Theorem B: this was done on sheet 9.

As always, thank you for the marvelous notes!

**Comment by FlorianAdler** [comment]
As always, thank you for your feedback! I didn't adopt the following changes:

- I decided to keep "noetherianness" and "abelianness", because I doubt I'd be able to write "noetherianity" and "abelianity" consistently (and at least they don't seem wrong). The other two though I changed as suggested.

- About s_\ell: I disagree. s_\ell\psi has indices i_0, ... , i_{n-1}, whereas \psi has n+1 indices (i_\ell being doubled), so they really should be elements of C^{n-1} and C^n respectively. I hope I'm not messing this up because this kind of stuff always gives me a headache.

- I looked through every instance of \Hom in Section 3 and for me they all seem correct. When I write \Hom_{O_C}(V,\Omega_C) instead of sheaf-Hom, this actually means the set (or group or _k_-vector space) of (O_C-linear) morphisms of sheaves between V and \Omega_C -- i.e. the global section of the sheaf-Hom. Whenever \Hom of two sheaves occurs in this section, it comes with a \dim_k or it is compared to H^1 of something (I hope, I didn't overlook anything), so it really wouldn't make sense to have a sheaf there instead of a group/_k_-vector space.
